qemu/tests/zero_copy.py

```diff
@@ -52,6 +52,9 @@ def enable_zerocopytx_in_host(enable=True):
         enable_zerocopytx_in_host(False)
 
     error.context("Boot vm with 'vhost=on'", logging.info)
+    if params.get("nettype") == "user":
+        raise error.TestNAError("Unable start test with user networking,"
```

Comment:
In existing tp-qemu tree, I only saw qemu/tests/drive_mirror_simple.py uses cancel() function. But I saw many tests still use error.TestNAError. Do we plan to fix existing tests that use error.TestNAError?